doc/classes/@GDScript.xml

```diff
@@ -348,6 +394,15 @@
 			<argument index="1" name="funcname" type="String">
 			</argument>
 			<description>
+				Returns a reference to the specified function 'funcname' in object 'instance'.
```

Comment:
`Returns a reference to the specified function 'funcname' in the 'instance' node. As functions aren't first-class objects in GDscript, use 'funcref' it to store a function in a variable and call it later.`